teamA/jeongHaeMan1/routes/postRouter.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express');
+const router = express.Router();
+
+const postController = require('../controllers/postController');
+
+router.post('/write', postController.write);
+router.get('/search', postController.search);
+router.patch("/modify", postController.modify);
+router.delete("/delete/:id", postController.deletePost);
```

Comment:
RESTful API에서 엔드포인트를 구축할 때 url에 동사가 아닌 명사형으로 작성을 해줘야 합니다.

RESTful 세션 이후 자료 참고하여 수정해주세요!

```suggestion
router.post('/', postController.write);
router.get('/', postController.search);
router.patch("/:postId", postController.modify);
router.delete("/:postId", postController.deletePost);
```